packages/docs/page-config/ui-elements/skip-link/index.ts

```diff
@@ -0,0 +1,20 @@
+import apiOptions from "./api-options";
+import apiDescription from './api-description';
+
+export default definePageConfig({
+  blocks: [
+    block.title("Skip Link"),
+    block.paragraph("`va-skip-link` skip to the target component."),
+
+    block.subtitle("Basic usage"),
+
+    block.paragraph("By default, `va-skip-link` needs a target and position, please keep in mind that the component should be placed as the very first element."),
```

Comment:
Can we add a short code example here? Just code. Right now it's not clear at all how to use component.